README.md

```diff
@@ -1,6 +1,7 @@
 # `dune-plot-style` - DUNE official plot styling tools
 
 This repository contains coding tools to help analyzers easily make plots adhering to the DUNE Plot Style documented at https://wiki.dunescience.org/wiki/DUNE_Plot_Styles.
+(Those guidelines are also duplicated in this repository as [`guidelines.md`](https://github.com/DUNE/dune-plot-style/blob/main/guidelines.md).)
 
```

Comment:
I would say to just keep the guidelines in github and link from elsewhere to here. Duplicated sets of guidelines are bound to get out of step with each other and it'll be a headache for people to figure out which is newer.

Reply:
Yeah, ok, that's a good call.

Reply:
And it would also be possible to directly publish the guidelines from there, as a nicer static website.

Reply:
eliminated the text about duplication in db323cf.